### DISCUSSION TITLE
Want to integrate with WHMCS

### DISCUSSION THREAD

**Discussion by mhktanbir** [chatwoot/chatwoot#5729] [Feature Requests]
Hi,

Can we integrate it with WHMCS system?
Bcz when someone wants about support then we can see her support ticket service etc from Chatwoot.

It can be helpful for provide faster support.

**Comment by pranavrajs** [comment]
There is a community plugin for WHMCS, please see if it fits. https://marketplace.whmcs.com/product/5530-chatwoot-whmcs-add-on

**Reply by mhktanbir** [reply]
> There is a community plugin for WHMCS, please see if it fits. https://marketplace.whmcs.com/product/5530-chatwoot-whmcs-add-on

It's just a module to add widgets on the website.

But I want something else.

Like someone connect with agent, the agent can see the customers active products, ticket's etc from the Chatwoot dashboard.

**Reply by pranavrajs** [reply]
Is it possible to build an integration using Dashboard Apps? https://www.chatwoot.com/docs/product/others/dashboard-apps/

**Reply by mhktanbir** [reply]
> Is it possible to build an integration using Dashboard Apps? https://www.chatwoot.com/docs/product/others/dashboard-apps/

How can I learn more about it?
